README.md

```diff
@@ -32,11 +33,11 @@ a commercial set of Instagram-like filters.
 
 ## How it works
 
-This method is based on
+The accurate color reconstruction method is based on
 [three-dimensional lookup tables][wiki-luts] and their two-dimensional
 representation: [hald images][hald-image].
 The core idea is simple: a sample hald image with a uniform color distribution
-is processed using any target software with an unknown color
+is processed using any target software color filter with an unknown
```

Comment:
"software color filter" looks like an opposite to "hardware color filter". While "transformation algorithm" is really "color transformation algorithm". Maybe this:

> is processed using target color filter with an unknown transformation algorithm.

?

Reply:
yup, looks good to me (while many out there know a hardware color filter is a physical colored thing you attach to your camera lens to adjust images)